packages/react-components/package.json

```
@@ -52,6 +52,7 @@
     "@fluentui/react-theme": "9.0.0-alpha.23",
     "@fluentui/react-tooltip": "9.0.0-alpha.85",
     "@fluentui/react-utilities": "9.0.0-alpha.49",
+    "@fluentui/react-text": "^9.0.0-alpha.18",
```

we are using pinned versions while in prerelease

```suggestion
    "@fluentui/react-text": "9.0.0-alpha.18",
```

---

That is what I did at first, but I got this issue when using the fixed version:
`✕ @fluentui/react-text
- ^9.0.0-alpha.18 in dependencies of vr-tests
 - 9.0.0-alpha.18 in dependencies of @fluentui/react-components
- ^9.0.0-alpha.18 in devDependencies of @fluentui/react-card`

Do you know how to get this fixed? Should I remove the caret on the `vr-tests` and `devDependencies`?

---

`yarn nx workspace-generator migrate-fixed-versions` should solve all those problems